services/src/datasets/external/netcdfcf/mod.rs

```diff
@@ -502,45 +502,89 @@ struct DimensionSizes {
     pub lon: usize,
 }
 
-#[derive(Debug)]
-struct GroupPath<'g> {
-    pub prefix: Vec<GroupPathSegment>,
-    pub current: MdGroup<'g>,
+#[derive(Debug, Clone, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct NetCdfOverview {
+    pub file_name: String,
+    pub title: String,
+    pub spatial_reference: SpatialReferenceOption,
+    pub subgroups: Vec<NetCdfSubgroup>,
```

Comment:
why is the field and the datatype called Subgroup and not Group?

Reply:
555f45e